_pkgdown.yml

```diff
@@ -1,6 +1,7 @@
 url: https://pharmaverse.github.io/admiraldev
 
 template:
+  bootstrap: 5
   params:
     bootswatch: flatly
     docsearch:
```

Comment:
BTW @bms63 where are you getting these API keys from? Do you _want_ to use docsearch? Seems unnecessary for such a small site, especially since BS5 gives you search out of the box.

Reply:
I think it should be updated to this .  I just tried the search and it doesn't work.   I remember there being an issue preivously with admiral for this docsearch thing.

![image](https://user-images.githubusercontent.com/10111024/190001696-d2c1bcc4-d49d-4b0a-bd07-e0290f28a202.png)

Reply:
Ok, yeah, I figured since docsearch is old news after BS5.